packages/itwinui-css/src/menu/menu.scss

```diff
@@ -32,6 +32,12 @@ $iui-active-outline: thin solid var(--iui-color-border-accent);
     overflow-y: overlay;
     --iui-menu-max-height: calc((var(--iui-component-height) - 1px) * 8.5); // Max 8.5 items in .iui-scroll
   }
+
+  &.iui-flat {
```

Comment:
This `flat` variant looks unrelated to the whole multi-level menu. Might not belong in this PR.